javascript/javascript3/week2/fetchWaiting.js

```diff
@@ -0,0 +1,33 @@
+//Fetching and waiting
+const url =
+    "https://gist.githubusercontent.com/pankaj28843/08f397fcea7c760a99206bcb0ae8d0a4/raw/02d8bc9ec9a73e463b13c44df77a87255def5ab9/movies.json";
+
+function waitingTime() {
+    console.log("Please wait for 3 seconds.");
+    return new Promise(function(resolve) {
+        setTimeout(() => {
+            resolve();
```

Comment:
There is an error regarding the call to `resolve`. One argument needs to be passed to it.
`Expected 1 arguments, but got 0. Did you forget to include 'void' in your type argument to 'Promise'?ts(2794)`